core/src/main/java/io/questdb/cairo/wal/WalPurgeJob.java

```diff
@@ -396,7 +396,7 @@ private long getSafeToPurgeUpToTxn(long readerSeqTxn) {
             final TableToken viewToken = childViewSink.get(v);
             final MatViewState state = engine.getMatViewStateStore().getViewState(viewToken);
             if (state != null && !state.isPendingInvalidation() && !state.isInvalid() && !state.isDropped()) {
-                final long appliedToViewTxn = Math.max(1, state.getLastRefreshBaseTxn());
+                final long appliedToViewTxn = Math.max(state.getLastRefreshBaseTxn(), 0);
```

Comment:
Manual refresh views should be excluded from consideration. Otherwise, it will create a WAL disk leak; MANUAL views may not be refreshed ever.